packages/cli/src/bin/pipcook-job.ts

```diff
@@ -18,6 +18,7 @@ program
   .option('--output <dir>', 'the output directory name', 'output')
   .option('-h|--host-ip <ip>', 'the host ip of daemon')
   .option('-p|--port <port>', 'the port of daemon')
+  .option('-c|config <config file>', 'the config file for this job')
```

Comment:
`pipcook run <id> -c foobar.json` looks like weird, we should only append the parameters only, not the complete config file.

Reply:
The config here refers to a param config file.

Reply:
I'm confused that, shall we have just command line parameter for that?

Reply:
The job param might be extremely long. In this case, putting params in CLI may not be the desired way.

Reply:
Hmm, but maintaining 2 configs is too confused for our end-users.